src/stcal/ramp_fitting/likely_fit.py

```diff
@@ -0,0 +1,1335 @@
+#! /usr/bin/env python
+
+import logging
+import multiprocessing
+import time
+import sys
+import warnings
+
+from multiprocessing import cpu_count
+from pprint import pprint
+
+import numpy as np
+
+from . import ramp_fit_class, utils
+from .likely_algo_classes import IntegInfo, Ramp_Result, Covar
+
+
+DELIM = "=" * 80
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+BUFSIZE = 1024 * 300000  # 300Mb cache size for data section
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+def likely_ramp_fit(
+    ramp_data, buffsize, save_opt, readnoise_2d, gain_2d, weighting, max_cores
```

Comment:
I think the jump threshold should be exposed to the user in the same way as it is for ordinary jump detection.  I suggest passing a parameter I will call `nsig` that is the same one input to ordinary jump detection.  Implementation suggestions are in a separate comment below.

Reply:
I will add the `nsig` to the `RampData` class and make appropriate updates to the code for signal threshold.  To change the parameter to be accessed through the pipeline requires further discussion because it changes function signatures to the step API for ramp fitting.  In particular, due to the `1/f` step that is currently in development requires jump detection run before ramp fitting.  Right now, for the likelihood algorithm runs the jump detection and ramp fitting together.

Reply:
Ok, here is my suggestion then (tagging @drlaw1558).  Leave the current jump detection before 1/f; it should be plenty good enough for those purposes.  If we want jump detection in here to be fully independent, then I think we want the option to *not* propagate the previously identified jumps into this step.  That would be a boolean argument that would ultimately be passed to `determine_diffs2use` where it would turn off the line 
`d2use[jmp_locs[1:, :] == 1] = 0`.  The jump detection threshold could remain the same; I don't think anything else would need to change.